commcare-cloud-bootstrap/environment/app-processes.yml.j2

```diff
@@ -1,15 +1,13 @@
 celery_processes:
   {{ celery_host.name }}:
     beat: {}
-    celery_periodic:
-      concurrency: 4
-    async_restore_queue,background_queue,case_rule_queue,celery,export_download_queue:
+    background_queue,case_rule_queue,celery,export_download_queue:
       concurrency: 1
     email_queue,reminder_case_update_queue:
       concurrency: 1
-    reminder_rule_queue,reminder_queue,repeat_record_queue,saved_exports_queue:
+    reminder_rule_queue,repeat_record_queue,saved_exports_queue:
       concurrency: 1
-    sms_queue,submission_reprocessing_queue,ucr_indicator_queue,ucr_queue:
```

Comment:
Is it worth adding these back in commented-out form? Since this has already been used for spinning up test servers, it'd be nice if at least all you had to do to get these back was uncomment some lines.

Reply:
Can do. Is there any reason that these were split up the way that they were? If it was just for visual reasons, I can put all the commented out ones in the same line.

Reply:
We have it configured so that multiple queues can be consumed by the same process—so putting queue names together on the same lime, comma-separated, makes them all get processed in one process. We do that mainly to conserve memory.

So yeah, I think putting the commented out ones all on the same line comma separated is a reasonable idea. Then if someone uncomments them, they get a relatively memory-conserving default.

Reply:
cool, fixed here:
https://github.com/dimagi/commcare-cloud/pull/1959/commits/3b905872d069604b7c8196677340e9b9547748bd